src/SASjs.ts

```diff
@@ -682,18 +685,40 @@ export default class SASjs {
 
     const validationResult = this.validateInput(data)
 
+    // status is true if the data passes validation checks above
     if (validationResult.status) {
       if (config.serverType === ServerType.Sasjs) {
-        return await this.sasJsJobExecutor!.execute(
-          sasJob,
-          data,
-          config,
-          loginRequiredCallback,
-          authConfig,
-          extraResponseAttributes
-        )
+        /**
+         * When sending the JSON data object to SAS, it is first converted to 
+         * a set of specially formatted CSVs.  These are passed as multi-part
+         * form data (converted to input files in the backend SAS session by the
+         * API). When running outside of a browser, the FormData object is not 
+         * available. So in this case, a slightly different executor is invoked, 
+         * which is similar to the sas9JobExecutor.
+         */
+        if (typeof FormData === 'undefined') {
+          // cli invocation
+          return await this.sasJsJobExecutor!.execute(
```

Comment:
Currently this executor needs to be updated, as it is not passing the content as multi-part-form upload.

The following snippet can help:
```
let formData = new NodeFormData()
```